example/http/chat_assistant_example.sh

```
@@ -0,0 +1,100 @@
+
+# 1. Create a chat assistant
+echo -e "\n-- Create a chat assistant"
+CHAT_RESPONSE=$(curl -s --request POST \
+     --url "${HOST_ADDRESS}/api/v1/chats" \
+     --header 'Content-Type: application/json' \
+     --header "Authorization: Bearer ${API_KEY}" \
+     --data '{
+      "name": "My Assistant",
+      "llm_id": "deepseek-chat"
+      }')
+CHAT_ID=$(echo $CHAT_RESPONSE | jq -r '.data.id')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)